src/depot_type.h

```diff
@@ -13,6 +13,8 @@
 typedef uint16 DepotID; ///< Type for the unique identifier of depots.
 struct Depot;
 
+static const DepotID INVALID_DEPOT = 0xFFFF;
```

Comment:
Thinking about the static_assert, why is this not just defined as `= INVALID_STATION` ?

Reply:
Thanks for the feedback. DepotID and INVALID_DEPOT are now defined in terms of their station counterparts.